### PR TITLE
Add support for Scala 2.13.3

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -58,6 +58,10 @@ env:
     TRAVIS_JDK=adopt@1.8.0-222
   - SCALA_VERSION=2.13.2
     TRAVIS_JDK=adopt@1.11.0-4
+  - SCALA_VERSION=2.13.3
+    TRAVIS_JDK=adopt@1.8.0-222
```
Comment (ignasi35) [inline .travis.yml:62]: `-222` for any particular reason?

File: .travis.yml [hunk 2]
```diff
@@ -58,6 +58,10 @@ env:
...
+  - SCALA_VERSION=2.13.3
+    TRAVIS_JDK=adopt@1.11.0-4
```
Comment (ignasi35) [inline .travis.yml:64]: Same thing. Are we using `1.11.0-4` instead of `1.11.0-7` for any reason?

File: .travis.yml (hunk 1 above)
Comment (raboof) [reply .travis.yml:62]: No, I just stuck with the same version as for the others.

Comment (ignasi35) [reply .travis.yml:62]: SGTM. Let's bump the JDK versions elsewhere.

**Pull request merged**